assets/chezmoi.io/mkdocs.yml

```diff
@@ -350,11 +350,6 @@ markdown_extensions:
 - admonition
 - meta
 - pymdownx.details
-- pymdownx.snippets:
-    base_path:
-      - assets/chezmoi.io/snippets
-      - snippets
-    check_paths: true
```

Comment:
You didn't like the idea of using snippets or just wanted to write different descriptions there? I wanted to use snippets to deduplicate flags descriptions for commands as well.

Reply:
I wanted different descriptions there, and also want to use snippets to de-duplicate flag descriptions for commands.

The tricky might be getting the embedded Markdown parser to understand snippets.